README.md

```diff
@@ -52,11 +52,11 @@ defmodule MyApp.Guardian do
   use Guardian, otp_app: :my_app
 
   def subject_for_token(resource, _claims) do
-    to_string(resource.id)
+    {:ok, to_string(resource.id)}
```

Comment:
@hassox / @scrogson — what about allowing folks to return result directly?  It seems that a common simple solution would be:

```elixir
def resource_from_claims(%{"sub" => sub}), do: Repo.get(User, String.to_integer(sub))

def subject_for_token(resource, _claims), do: to_string(resource.id)
```

But as it is now, this is what we need:

```elixir
def resource_from_claims(%{"sub" => sub}) do
  case Repo.get(User, String.to_integer(sub)) do
    nil -> {:error, "not found"}
    resource -> {:ok, resource}
  end
end

def subject_for_token(resource, _claims), do: {:ok, to_string(resource.id)}
```